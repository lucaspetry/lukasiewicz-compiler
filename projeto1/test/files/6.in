int a = 0
a = a + true
