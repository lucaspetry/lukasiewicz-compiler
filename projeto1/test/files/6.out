[Line 3] semantic error: addition operation expected integer but received boolean
[Line 9] semantic error: addition operation expected boolean but received integer
[Line 9] semantic error: attribution operation expected float but received boolean
[Line 12] semantic error: addition operation expected boolean but received integer
[Line 12] semantic error: attribution operation expected integer but received boolean
[Line 13] semantic error: addition operation expected integer but received boolean
